packages/frontend-2/components/project/PendingFileImportStatus.vue

```diff
@@ -10,8 +10,8 @@
         ].includes(upload.convertedStatus)
       "
     >
-      <span class="text-body-xs mb-1">
-        {{ isSelfImport ? 'Importing' : 'Uploading new version' }}
+      <span class="text-body-xs mb-1 text-center">
+        {{ isSelfImport ? 'Processing upload' : 'Processing uploaded version' }}
```

Comment:
improved copy to better imply that the user does not have to sit there waiting for the "upload" to finish (cc @iainsproat )